src/backend/launcher.ts

```diff
@@ -119,16 +124,36 @@ async function prepareLaunch(
     gameSettings.useSteamRuntime &&
     (isNative || gameSettings.wineVersion.type === 'proton')
   if (shouldUseRuntime) {
+    // Determine which runtime to use based on toolmanifest.vdf which is shipped with proton
+    let nonNativeRuntime: SteamRuntime['type'] = 'soldier'
```

Comment:
is there any documentation on steam runtime soldier, sniper, scout? I'm not sure what this means

Reply:
Yes, on Valve's Gitlab page for Proton.